api/models/timelog.py

```diff
@@ -24,7 +24,7 @@ class Task(Base):
     story = Column(String(length=80), nullable=True)
     telework = Column(Boolean, nullable=True)
     text = Column(String(length=8192), nullable=True)
-    ttype = Column(String(length=40), nullable=True)
+    task_type = Column("ttype", String, ForeignKey("task_type.slug"), nullable=True)
```

Comment:
Probably silly question: why use task_type.slug as the foreign key instead of the task_type.id?

Reply:
This is to be a bit less disruptive with the current state. Since in the current column it already saves the slug the values would be pretty much the same.

I also thought that for queries that don't require the full name we wouldn't need join with the task type table in order to get the type of the task.

Reply:
If it makes more sense we could make the slug the primary key and remove the id column?

Reply:
Ah, of course! This makes sense. It'll work fine as-is; I think we could leave the id column in case we ever wanted to go another direction.